Package/Sublime Text Commands/Sublime Text Commands.sublime-syntax

```diff
@@ -68,6 +75,33 @@ contexts:
         3: punctuation.definition.string.end.json
       set: [expect-any-value, expect-colon]
 
+  expect-platform-string-value:
+    - match: (?=")
+      set: [mapping-value-meta, platform-name-string-pop]
+    - include: expect-string-rest
+
+  platform-name-string-pop:
+    - match: (\")(!)?(Windows|Linux|OSX)(\")
+      scope: meta.platform-name.sublime-commands string.quoted.double.json
+      captures:
+        1: punctuation.definition.string.begin.json
+        2: keyword.operator.logical.sublime-commands
+        3: support.constant.command-name.sublime-commands
+        4: punctuation.definition.string.end.json
+      pop: true
+    - match: \" # probably illegal command name
```

Comment:
Incorrect comment.

Reply:
This (& the below) is verbatim taken from the `.sublime-menu` syntax. I just changed the suffix scope to `.sublime-commands`. Does that mean it's wrong in `.sublime-menu` syntax ? (Perhaps I am mistaken)

Reply:
Ah yes, it is probably wrong there as well, since this isn't about command names.

Reply:
This is resolved. Removed it from both menu and command syntax. Added tests as well.